### PR TITLE
Fixed grammar error

### PR DESCRIPTION
**Pull request by Lagggpixel**

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/stats/StatsListener.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public void onArenaLeave(PlayerLeaveArenaEvent event) {
                 PlayerStatChangeEvent ev5 = new PlayerStatChangeEvent(player, event.getArena(), PlayerStatChangeEvent.StatType.DEATHS);
                 PlayerStatChangeEvent ev6 = new PlayerStatChangeEvent(damager, event.getArena(), PlayerStatChangeEvent.StatType.KILLS);
 
-                // killer is null if is he already received kill point.
+                // killer is null if he already received kill point.
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/stats/StatsListener.java]: ?!?! doesnt make sense

Comment (Lagggpixel) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/stats/StatsListener.java]: It makes very much a lot of sense.

**Review by tomkeuper — CHANGES_REQUESTED** [review]
pls fix

**Review by tomkeuper — APPROVED** [review]

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/stats/StatsListener.java (hunk 1 above)
Comment (Lagggpixel) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/stats/StatsListener.java]: Alright I've fixed.

**Review by tomkeuper — APPROVED** [review]

**Pull request closed without merging**